sdk/Javascript/src/Push/Push.FxOS.js

```diff
@@ -0,0 +1,413 @@
+// ----------------------------------------------------------------------------
```

Comment:
Sorry I was unclear.  This comment was meant in reference to Push.FxOs.js vs Push.Web.js.  Could we not just add the mdn support to Push.Web.js and use the same file for the FxOS & Web builds?  I don't see any other major difference here. 

Otherwise, does FireFox OS use any of these push systems?  It seems the FireFox OS version should only support mdn?  Why keep apns/gcm in it?